src/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ endif()
 # 3rd party lua sources
 lua_source(lua_sources ../third_party/luafun/fun.lua fun_lua)
 lua_source(lua_sources ../third_party/lua/luadebug.lua luadebug_lua)
+lua_source(lua_sources ../third_party/checks/checks.lua checks_lua)
```

Comment:
The .lua.c file is generated inside the `third_party/checks` directory and we can't ignore it using .gitignore in the parent repository. It makes the in-source build dirty:

```sh
$ cmake . -DCMAKE_BUILD_TYPE=Debug -DENABLE_BACKTRACE=ON -DENABLE_DIST=ON -DENABLE_FEEDBACK_DAEMON=ON -DENABLE_BUNDLED_LIBCURL=OFF && make -j
<...>
$ git status
HEAD detached at DifferentialOrange/gh-7726-embed-checks
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
	modified:   third_party/checks (untracked content)

no changes added to commit (use "git add" and/or "git commit -a")
$ git -C third_party/checks status
HEAD detached at 90bbf3e
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	checks.lua.c

nothing added to commit but untracked files present (use "git add" to track)
```

There are two ways to resolve it:

* Add `*.lua.c` into .gitignore into checks.
* Add `ignore = dirty` into .gitmodule in the parent repository.

I would prefer the first, but can accept the second.

Reply:
Added first in https://github.com/tarantool/checks/pull/50